Dockerfile

```diff
@@ -1,11 +1,49 @@
-FROM golang:1.19 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13 AS builder
```

Comment:
Just use Dockerfile.rhel7 for now.

Reply:
need golang 1.19 right ?

Reply:
also this is temporary change , once RHEL build with gpsd  and ubxtool ,we will change the docker file